docs/server-side-rendering.md

```diff
@@ -1,7 +1,7 @@
-Shared
+Server Side Rendering
 ======
 
-Contains any code that can be used both in a Calypso client (browser or desktop) and the Calypso Node server. Generally, any module in this folder will have either no client-specific dependencies (such as the DOM), or differing entry points for client and server specified in its `package.json` file, like this:
+Something something on the server.
```

Comment:
Any suggestions on what we want in this README, or is it fine as it is, with the addition of an intro?